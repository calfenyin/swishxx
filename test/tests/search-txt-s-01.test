search | | -s | absolute | 0
