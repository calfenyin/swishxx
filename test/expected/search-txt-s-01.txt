# results: 2
100 ./GNU_GPLv2.txt 17982 GNU_GPLv2.txt
37 ./Time_Machine,_The.txt 182203 Time_Machine,_The.txt
